### PR TITLE
introduce sandbox config

### PR DESCRIPTION
**Pull request by kramasamy**
if we have sandbox config - we can further reduce docker image size.

### CODE REVIEW

File: heron/config/src/yaml/BUILD [hunk 1]
```diff
@@ -23,6 +23,11 @@ filegroup(
 )
 
 filegroup(
+    name = "conf-sandbox-yaml",
```
Comment (huijunw) [inline heron/config/src/yaml/BUILD]: who will use these files `conf-sandbox-yaml`

Comment (kramasamy) [reply heron/config/src/yaml/BUILD]: @huijunw - this is for docker sandbox image.

Comment (huijunw) [reply heron/config/src/yaml/BUILD]: thanks @kramasamy

**Comment by nlu90** [comment]
👍

**Pull request merged**